src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -764,14 +773,17 @@ function parseAllOut(cue: string[]): CueDefinitionClearGrafiks {
 }
 
 export function parsePgmClean(cue: string[]): CueDefinitionPgmClean {
+	const pgmSource = cue[0].match(/^PGMCLEAN=(.+)$/i)
 	const pgmCleanCue: CueDefinitionPgmClean = {
 		type: CueType.PgmClean,
-		source: 'PGM',
-		iNewsCommand: 'PGMCLEAN'
+		iNewsCommand: 'PGMCLEAN',
```

Comment:
I don't know if this command is something we can change, but if we can it would be nice that it was 'PGM_CLEAN' instead of 'PGMCLEAN' :)